README.md

```diff
@@ -394,7 +463,7 @@ A very fast version of `time_filter()`. However, the request parameters are much
 * locations: List[Locations] - All locations. Location ids must be unique.
 * search_ids: Dict[str, List[str]] - Searches from a target location to destinations. You can define up to 100,000
   destinations
-* [transportation](#transportation): Union - Transportation mode and related parameters.
+* [transportation]: Transportation - Transportation mode and related parameters.
```

Comment:
fast endpoints have different transportation types, maybe I'll add a readme entry for them too